packages/node_modules/@ciscospark/internal-plugin-wdm/src/interceptors/server-error.js

```diff
@@ -0,0 +1,43 @@
+/*!
+ * Copyright (c) 2015-2017 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {Interceptor} from '@ciscospark/http-core';
+import {SparkHttpError} from '@ciscospark/spark-core';
+/**
+ * Changes server url when it fails
+ */
+export default class ServerErrorInterceptor extends Interceptor {
+  /**
+  * @returns {HAMessagingInterceptor}
+  */
+  static create() {
+    // eslint-disable-next-line no-invalid-this
+    return new ServerErrorInterceptor({spark: this});
+  }
+
+
+  /**
+   * @see Interceptor#onResponseError
+   * @param {Object} options
+   * @param {Object} reason
+   * @returns {Object}
+   */
+  onResponseError(options, reason) {
+    const feature = this.spark.internal.device.features.developer.get('web-ha-messaging');
+    if (feature && feature.value) {
+      if (options.uri) {
```

Comment:
Any reason not combine this with the line above?

Reply:
I thought the same thing but went with Tran's work to keep the changes minimal. But better code is a better reason than inertia lol. I'll make the change.